sympy/diffgeom/diffgeom.py

```diff
@@ -86,25 +114,34 @@ class Patch(Atom):
     Examples
     ========
 
-    Define a Manifold and a Patch on that Manifold:
-
     >>> from sympy.diffgeom import Manifold, Patch
-    >>> m = Manifold('M', 3)
+    >>> m = Manifold('M', 2)
     >>> p = Patch('P', m)
-    >>> p in m.patches
-    True
+
+    >>> p.name
+    'P'
+    >>> p.dim
+    2
+
+    References
+    ==========
+
+    .. [1] G. Sussman, J. Wisdom, W. Farr, Functional Differential Geometry (2013)
 
     """
-    # Contains a reference to the parent manifold in order to be able to access
-    # other patches.
-    def __new__(cls, name, manifold):
+    def __new__(cls, name, manifold, **kwargs):
```

Comment:
Does this need any keyword arguments?

Reply:
Okay, you may have kept in mind that some sympy keywords like `evaluate` can propagate down

Reply:
You got it right.